### PR TITLE
Docker baseline type config

### PR DESCRIPTION
**Pull request by Rybov**

### CODE REVIEW

File: Backend/src/api/movies.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                    videoURL: move.video_url,
+                    user: user,
+                };
+            })
+            .filter(move => !ratingMoveIds.includes(move.id))
+        return res.json({ moves: moves });
+    } catch (error) {
+        return res.status(400).json({ error: "Bad request" });
+    }
+});
```
Comment (fatihkaganemre) [inline Backend/src/api/movies.js:29]: I think i already have this file, btw Moves not movies

File: Backend/src/api/ratings.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        });
+        return res.json( {ratings: ratings} )
+    } catch (error) {
+        return res.status(400).json({ error: "Bad request" });
+    }
+});
+
+ratingsRoutes.get('/health', async (req, res) => {
+    return "ok"
+})
```
Comment (fatihkaganemre) [inline Backend/src/api/ratings.js:48]: I think i also have this file

**Review by fatihkaganemre — CHANGES_REQUESTED** [review]
should we push node_modules, it was in git ignore ?  And please resolve conflicts. I wll wait for your merge and not code.

File: Backend/src/api/movies.js (hunk 1 above)
Comment (Rybov) [reply Backend/src/api/movies.js:29]: sure, i just move all api related files to api directory to have them in single place

**Comment by Rybov** [comment]
Noo, absolutly we should not store any node_modules in git repository, all code dependencies are discribed in package-lock.json

**Review by fatihkaganemre — APPROVED** [review]

**Pull request merged**